packages/nextjs-mf/src/loaders/nextPageMapLoader.ts

```diff
@@ -54,6 +63,15 @@ export function exposeNextjsPages(cwd: string) {
     };
```

Comment:
The function 'preparePageMap' can be improved by using a map function instead of a for loop. This can improve readability and maintainability.